www/result.inc

```diff
@@ -125,8 +125,15 @@ $page_description = "Website performance test result$testLabel.";
                     ?>
                 </div>
                 <?php
+                    $fvMedian = $testResults->getMedianRunNumber($median_metric, false);
+                    $rvMedian = $testResults->getMedianRunNumber($median_metric, true);
+
                     // create a friendlier (unique) name for the fi
                     $fileUrl = GetFileUrl($url);
+                    if ( $test['testinfo']['lighthouse'] && $fvMedian )
+                    {
+                      echo "<a href=\"/lighthouse.php?test=$id&run=$fvMedian\">Lighthouse report</a><br>";

```

Comment:
This works for now but I'd really like to give it much more prominent treatment.  The grades section in the header is a perfect place since the grades won't be calculated in the lighthouse case.  If the score can be pulled out of the HTML then the UI can have a large:

Lighthouse Score: 85/100 (click for full report)

And maybe also the logo ;-)